src/Chainweb/Chainweb.hs

```diff
@@ -512,3 +516,6 @@ runChainweb cw = do
   where
     logg = logFunctionText $ _chainwebLogger cw
     mempoolP2pConfig = _configMempoolP2p $ _chainwebConfig cw
+
+    loggServerError (Just r) e = "HTTP server error: " <> sshow e <> ". Request: " <> sshow r
+    loggServerError Nothing e = "HTTP server error: " <> sshow e
```

Comment:
Is this for debugging or prod usage?

Reply:
It is for production usage. The default behavior of wrap is to print any exception from the application to `stderr`. This emits on error log.